library/ix-dev/community/mumble/templates/_mumble.tpl

```diff
@@ -0,0 +1,119 @@
+{{- define "mumble.workload" -}}
+workload:
+  mumble:
+    enabled: true
+    primary: true
+    type: Deployment
+    podSpec:
+      hostNetwork: false
+      containers:
+        mumble:
+          enabled: true
+          primary: true
+          imageSelector: image
+          securityContext:
+            runAsUser: 1000
+            runAsGroup: 1000
+          envFrom:
+            - configMapRef:
+                name: mumble-config
+            - secretRef:
+                name: mumble-secret
+          {{ with .Values.mumbleConfig.additionalEnvs }}
+          envList:
+            {{ range $env := . }}
+            - name: {{ $env.name }}
+              value: {{ $env.value }}
+            {{ end }}
+          {{ end }}
+          probes:
+            # Probes are disabled because it fills the logs with
+            # "connection attemps"
+            liveness:
+              enabled: {{ .Values.ci }}
+              type: tcp
+              port: "{{ .Values.mumbleNetwork.serverPort }}"
+            readiness:
+              enabled: {{ .Values.ci }}
+              type: tcp
+              port: "{{ .Values.mumbleNetwork.serverPort }}"
+            startup:
+              enabled: {{ .Values.ci }}
+              type: tcp
+              port: "{{ .Values.mumbleNetwork.serverPort }}"
+      initContainers:
+      {{- include "ix.v1.common.app.permissions" (dict "containerName" "01-permissions"
+                                                        "UID" 1000
+                                                        "GID" 1000
+                                                        "mode" "check"
+                                                        "type" "init") | nindent 8 }}
```

Comment:
I think this meant that each time container start it will check parent and then chown..is that required?

Reply:
It will only chown if the uid in the parent have changed.
If we assume user won't change the permissions on the hostpath via UI/shell or other means, it's not needed.
Let's change it to only run on install once, considering that they UID is hardcoded and not configurable?

Reply:
Sounds good to me ✅

Reply:
Done